cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/PushCommandTest.java

```diff
@@ -71,12 +71,28 @@ public void setup() throws IOException {
     public void testPushWithInvalidPath() throws IOException {
         Path validBalProject = this.testResources.resolve(VALID_PROJECT);
         PushCommand pushCommand = new PushCommand(validBalProject, printStream, printStream, false);
-        new CommandLine(pushCommand).parse("tests");
+        String invalidPath = "tests";
+        new CommandLine(pushCommand).parse(invalidPath);
         pushCommand.execute();
 
         String buildLog = readOutput(true);
         String actual = buildLog.replaceAll("\r", "");
-        String expected = "file provided is not a bala file.";
+        String expected = "path provided for the bala file does not exist: " + invalidPath + ".";
```

Comment:
If we think of this from a more user perspective,
1. User will run `bal pack` command
2. After completion user will run `bal push`
3. Meantime if somehow bala is not in the `target/bala` folder, This error happens.
`path provided for the bala file does not exist: target/bala/winery-lib-1.0.0.bala`
Since the user is not giving any path, is this the best error message we can show?
@azinneera WDYT?

Reply:
Seems this error only shows when the user has given the bala path.
Hense resolving the above comment.